assets/js/meta-box-resize.js

```diff
@@ -1,47 +0,0 @@
-( function() {
```

Comment:
If this folder no longer exists, it should be removed from `.eslintignore` and `build-plugin-zip.sh`:

https://github.com/WordPress/gutenberg/blob/4da6c0f317a148ad87ce630c327abb4e3fb498d3/.eslintignore#L5
https://github.com/WordPress/gutenberg/blob/4da6c0f317a148ad87ce630c327abb4e3fb498d3/bin/build-plugin-zip.sh#L96